packages/datagateway-search/src/table/investigationSearchTable.component.tsx

```diff
@@ -227,8 +227,7 @@ const InvestigationSearchTable = (
         return (
           <div>
             <Typography>
-              <b>{t('investigations.rb_number')}:</b>{' '}
-              {investigationData.rbNumber}
+              <b>{t('investigations.name')}:</b> {investigationData.name}
```

Comment:
Careful here. I don't think this reference actually exists in datagateway-search's `res/default.json`. `rb_number` is there though, so it may be worth replacing `rb_number` with `name`

Reply:
Thanks, @sam-glendenning, I had overlooked the `res/default.json` file. Made the changes now.